### PR TITLE
Add IBM 3rd Party Tools and Fix PR link

### PR DESCRIPTION
**Pull request by marcchua**
Added 3rd party tools vault-cli and vault-go per IBM's request. 
Fixed the PR link to reference main instead of master, which was causing a 404.

### CODE REVIEW

**Review by taoism4504 — APPROVED** [review]
LGTM

**Comment by VioletHynes** [comment]
Hey! Thanks for this PR. I fixed the conflict, and going to get this one merged! Apologies it took a little while!

File: website/content/api-docs/relatedtools.mdx [hunk 1]
```diff
@@ -32,6 +32,8 @@ The following list of tools is maintained by the community of Vault users; Hashi
 - [Cruise Daytona](https://github.com/cruise-automation/daytona) - An alternative implementation of the Vault client CLI for services and containers. Its core features are the ability to automate authentication, fetching of secrets, and automated token renewal. Support for AWS, GCP, & Kubernetes Vault Auth Backends.
 - [Vault-CRD](https://vault.koudingspawn.de/) - Synchronize secrets stored in HashiCorp Vault to Kubernetes Secrets for better GitOps without secrets stored in git manifest files.
 - [vsh](https://github.com/fishi0x01/vsh) - Interactive shell with tab-completion. Allows recursive operations on paths. Allows migration of secrets between both KV versions.
+- [vault-cli](https://github.com/IBM/vault-cli) - A yaml based automation tool that bootstraps vault cluster(s) with the desired configuration (namespaces, endpoints, policies, roles, endpoint)
```
Comment (schavis) [inline website/content/api-docs/relatedtools.mdx:35]: Vault should be capitalized. Otherwise LGTM

**Review by schavis — DISMISSED** [review]

File: website/content/api-docs/relatedtools.mdx (hunk 1 above)
Comment (VioletHynes) [inline website/content/api-docs/relatedtools.mdx:35]:
```suggestion
- [vault-cli](https://github.com/IBM/vault-cli) - A yaml based automation tool that bootstraps Vault cluster(s) with the desired configuration (namespaces, endpoints, policies, roles, endpoint)
```

Comment (VioletHynes) [reply website/content/api-docs/relatedtools.mdx]: I fixed this for the submitter -- feel free to re-review!

**Pull request merged**